pyam/plotting.py

```diff
@@ -274,7 +274,7 @@ def pie(
     value="value",
     category="variable",
     legend=False,
-    title=True,
+    title=False,
```

Comment:
Here I would suggest a default value of `None`. In other plotting funcs, title can be `True` if there is logic available to find a reasonable default. If not, then it will be set by the kwarg value. Would suggest also updating docstring to `string, optional`

Reply:
Done - also change the docstring for the `title` keyword argument to follow [Axes.set_title()](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.axes.Axes.set_title.html)